src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderDatePicker.vue

```diff
@@ -24,11 +24,13 @@
 	<div class="datepicker-button-section">
 		<button v-shortkey="previousShortKeyConf"
 			:aria-label="previousLabel"
-			class="datepicker-button-section__previous button icon icon-leftarrow"
+			class="datepicker-button-section__previous button"
 			:title="previousLabel"
 			type="button"
 			@click="navigateToPreviousTimeRange"
-			@shortkey="navigateToPreviousTimeRange" />
+			@shortkey="navigateToPreviousTimeRange">
+			<ChevronLeftIcon size="22" />
```

Comment:
for educational purposes, is there any specific reason why we do 22 here the size and 20 for the normal action icons?

Reply:
I increased the size here a little to make it look like it has before. The old arrow icons were quite big.